rules/eslint/style/on.js

```diff
@@ -25,7 +25,7 @@ module.exports = {
     // enforce newline at the end of file, with no multiple empty lines
     "eol-last": "error",
     // require or disallow spacing between function identifiers and their invocations
-    "func-call-spacing": "off",
+    "func-call-spacing": ["error", "never"],
```

Comment:
I didn't even think that you could do this. It makes sense but what horrible code that would be...